client/public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="rtl">
```

Comment:
is it possible to use lang = ar?
[link](https://www.w3schools.com/tags/ref_language_codes.asp)